data-prepper-plugins/neptune-source/src/main/java/org/opensearch/dataprepper/plugins/source/neptune/client/NeptuneDataClientWrapper.java

```diff
@@ -0,0 +1,112 @@
+/*
+ *
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
```

Comment:
Did you copy code from another project into this file? If not, you can use the simple two-line header. If so, this is fine.